app/src/main/java/dev/andrylat/raqimbek/bankingutils/BankingUtilsApp.java

```diff
@@ -0,0 +1,26 @@
+package dev.andrylat.raqimbek.bankingutils;
+
+public class BankingUtilsApp {
+    private static CardValidationUserInteraction userInteraction = new CardValidationUserInteraction();
+    private static CardValidator cardValidator = new CardValidator();
+    private static PaymentSystemDeterminer paymentSystemDeterminer = new PaymentSystemDeterminer();
+
+    public static void main(String[] args) {
+        run();
+    }
+
+    public static void run() {
+    	userInteraction.write("Hello. Enter card number for validation:", System.out);
+    	userInteraction.read();        
+
+    	var cardValidationInfo = cardValidator.checkCardNumber(userInteraction.getInput());
+
+    	if (cardValidationInfo.getErrors().size() > 0) {
+    	    userInteraction.write(cardValidationInfo.errors.toString(), System.out);
```

Comment:
".errors.". Nope. 

Always take the value only thought a method. Why? Because in other case I can clear "errors" from random place of the code and debugging of such kind of errors is very complex. Method can control what will be done.

Reply:
Also user interaction class should receive input and output in constructor. You don't need to define where output should be placed each time. It is not easy to use.